build.sh

```diff
@@ -2,10 +2,11 @@
 set -euo pipefail
 
 bail="--bail"
+run_tests="true"
 while [[ "${1:-}" != "" ]]; do
     case $1 in
         -h|--help)
-            echo "Usage: build.sh [--no-bail] [--force|-f]"
+            echo "Usage: build.sh [--no-bail] [--force|-f] [--skip-test]"
```

Comment:
Should probably be called "--no-test" instead (to match "--no-bail").

Reply:
Not sure that's the same thing.

Reply:
I'll leave `--skip-test[s]` if that's okay